### DISCUSSION TITLE
Engineering Job Opportunity @ Den

### DISCUSSION THREAD

**Discussion by isvidler** [smartcontractkit/full-blockchain-solidity-course-py#1729] [General]
Hey everyone,

@PatrickAlphaC suggested I share our job opportunity with you all.

We're hiring software engineers over at Den. If you have experience as a web2 dev and want to transition to crypto, you might be a good fit!

I'll keep this post light as the job description has all the details. You can find it here: https://onchainden.com/hiring

A couple notes:
• The role is in-person in NYC
• We're currently not sponsoring visas

If you're not a good fit but know someone who is, we're sending $2,000 USDC to anyone that introduces us to engineers we end up hiring. The payment's conditional on > 6 months tenure. If you know someone who fits the criteria, reach out to me on Twitter (https://twitter.com/IttaiSvidler)!

**Comment by cromewar** [comment]
Pinned this discussion ;)
